### ISSUE TITLE
Beyond module keeps crashing

### ISSUE THREAD

**Issue by corposcr** [bitfocus/companion-module-pangolin-beyond#1]
Hello,
               Hope all is well. I wanted to let you know that the module for Beyond keeps on crashing when executing any action. I'm using windows 10, Companion 2.1.3 and Beyond 5 latest release. 

Not a biggie, I.m building my own interface with OSC and pangoscripts. 
![image](https://user-images.githubusercontent.com/94130676/141754699-628d5460-495b-482e-9fc9-8d80f60831de.png)

**Comment by krocheck** [comment]
Hi @corposcr.  I was able to replicate this if the "Target Port" field in the configuration isn't filled in.  I filled in 4324 as a test and the errors stopped.  You'd have to fill this in with whatever port your configuration is set to.  I found one reference [here](https://blt-guide.deepsymmetry.org/beat-link-trigger/0.6.3/integration_beyondadvanced) that the port is 16062.  Let me know if, in fact, that is the default and we can program that way for a future release.

![image](https://user-images.githubusercontent.com/749812/141779838-b4b51b36-b38e-46b9-b460-96577282702f.png)
